store/sqlstore/store_test.go

```diff
@@ -1,151 +0,0 @@
-// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Renamed this file to `storetest.go` so that its `Setup()` method is available on the test classpath